### ISSUE TITLE
[CNTOOLS] Change default for currentKESPeriod() to manual calculation

### ISSUE THREAD

**Issue by rdlrt** [cardano-community/guild-operators#427]
Some peepz used kes keys from different network and used it against mainnet. For such cases rotate KES does not work.
In cntools.library, replace the function below:

``` bash
getCurrentKESperiod() {
  getPromMetrics
  current_kes_period=$(grep "cardano_node_Forge_metrics_currentKESPeriod_int" "${TMP_FOLDER}"/prom_metrics | awk '{print $2}')
  if [[ -z "${current_kes_period}" ]]; then
    # lets calculate it instead
    current_kes_period=$(( $(getSlotTip) / slots_per_kes_period ))
  fi
  echo "${current_kes_period}"
}
```

with:

``` bash
getCurrentKESperiod() {
  current_kes_period=$(( $(getSlotTip) / slots_per_kes_period ))
  echo "${current_kes_period}"
}
```

(alongwith editing minor version in cntools.library and cntools-changelog.md)

**Comment by rdlrt** [comment]
@Dostrelith [EDEN] picked his hand up to add this 😄

**Comment by rdlrt** [comment]
Closed by #428
